### DISCUSSION TITLE
Whisper producing differing results and different inference time at every inference

### DISCUSSION THREAD

**Discussion by nome2050** [openai/whisper#913] [Q&A] [answered]
I am using t4 gpu to run whisper medium en

the code sometimes gives different results at different inference

also, the time is also variable

that is making the benchmark process difficult

also, any way to decrease inference time

**Comment by jongwook** [comment, marked as answer]
Please see #81 for the nondeterminism. Some avenues for optimization includes quantization and CUDA kernel fusion, which has been discussed in #454 and #115, but are not yet included in this repo.
